deel-1-flexbox-positioneren/styles/styles.css

```diff
@@ -12,20 +12,32 @@
 .opdracht-2 ul{
```

Comment:
If you need to use the same selector more than once you can use a class selector here rather than the ul element.